### DISCUSSION TITLE
How to add column to public.users

### DISCUSSION THREAD

**Discussion by razvanh** [supabase/supabase#13247] [Questions]
Playing with [Next.js Subscription Payments Starter](https://github.com/vercel/nextjs-subscription-payments) and stumbled across an issue. The api/integration is creating the project and adding the tables when deploying to Vercel. The tables created are owned by `supabase_admin`, even those in the `public` schema. Now I'm trying to add a new column to the `public.users` table using the UI but I get the following error: `failed to update pg.tables with the given ID: must be owner of table users`.  There is no way to transfer ownership to the `postgres` user, or to grant the `postgres` user privileges to alter that table.

**Comment by razvanh** [comment]
Contacted support and they updated the perms for me

**Comment by bmeneghini** [comment]
I'm facing the same issues, this is super annoying. We should be able to edit the tables with the default policy.

**Comment by louis030195** [comment]
still no solution? do we have to contact support every time we use a template?
